mlibrary/catalog-browse
# [SEARCH-1604] Implement design for author browse

# Overview
The fake data is up for `Browse by author`, and the site needs help in displaying it similar to `Browse by call number (LC and Dewey)`.

This pull request resolves [SEARCH-1604](https://tools.lib.umich.edu/jira/browse/SEARCH-1604).

## Anything else?
### Layout
`./views/layout.erb` has been broken up so it now it acts a template for other browse options.

### CSSNano
`cssnano` has been added to help minify the built stylesheet so it doesn't get bloated with all of the documentation.

## Testing
- Run the tests to make sure they pass (`docker-compose run --rm web bundle exec rspec`).
- Build the CSS (`docker-compose run --rm web npm run build`).
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Go to [Browse by author](http://localhost:4567/author) and see...
  - If the information is displayed.
  - The website doesn't look borked.
